src/components/search/detailed/DataSearchResults.js

```diff
@@ -149,21 +143,25 @@ function DataSearchResults(props) {
                 DATA_SEARCH_QUERY_KEY,
                 {
                     searchTerm,
-                    userHomeDir,
+                    userHomeDir: "",
                     rowsPerPage: searchConstants.DETAILED_SEARCH_PAGE_SIZE,
                     sortField: sortField,
                     sortDir: sortOrder,
                 },
             ]);
             setDataSearchQueryEnabled(true);
         }
-    }, [searchTerm, sortField, sortOrder, userHomeDir]);
+    }, [searchTerm, sortField, sortOrder]);
```

Comment:
Now I think about it - this still doesn't cover 1 use case where there are no search results matching the the term entered by users. In that case we don't show the `See all results for `term` option in the drop-down. So user has to hit `enter` to see all the results. May be we should that option even when there are no matching top 10 results for `Data` 🤔

Reply:
Sounds like a good idea to me 👍

Reply:
Ok this done.